RecordId should be public

***tl;dr*** make `type RecordId` public so it can be `use`d

I'd like to store the `RecordId` instance for later processing. While I know the `RecordId` is a `u64`, it would nice if I could simply refer to `RecordId`.

As in this contrived example.
```lang-rust
use ::evtx::EvtxParser;
use ::evtx::RecordId;

fn main() {
    let fp = PathBuf::new();
    let mut parser = EvtxParser::from_path(fp).unwrap();
    let mut ids: Vec<RecordId> = vec![];
    for record in parser.records() {
        match record {
            Ok(r) => {
                ids.push(r.event_record_id);
            },
            _ => {},
        }
    }
}

```

Currently, that code does not compile

```lang-text
error[E0432]: unresolved import `evtx::RecordId`
 --> src\main.rs:2:5
  |
2 | use ::evtx::RecordId;
  |     ^^^^^^^^^^^^^^^^ no `RecordId` in the root
```

Using `evtx` version 0.8.1.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -13,7 +13,7 @@ extern crate bitflags;
 pub use evtx_chunk::{EvtxChunk, EvtxChunkData, EvtxChunkHeader, IterChunkRecords};
 pub use evtx_file_header::{EvtxFileHeader, HeaderFlags};
 pub use evtx_parser::{EvtxParser, IntoIterChunks, IterChunks, ParserSettings};
-pub use evtx_record::{EvtxRecord, EvtxRecordHeader, SerializedEvtxRecord};
+pub use evtx_record::{EvtxRecord, EvtxRecordHeader, RecordId, SerializedEvtxRecord};
 pub use json_output::JsonOutput;
 pub use json_stream_output::JsonStreamOutput;
 pub use xml_output::{BinXmlOutput, XmlOutput};

--- a/tests/test_record_id_public.rs
+++ b/tests/test_record_id_public.rs
@@ -1,0 +1,8 @@
+use evtx::RecordId;
+
+#[test]
+fn record_id_is_public() {
+    let id: RecordId = 42;
+    assert_eq!(id, 42);
+}
+

